Linear/main_LinEl.C

```diff
@@ -160,9 +162,20 @@ int main (int argc, char** argv)
         adaptor = atoi(argv[i]+5);
     }
     else if (!infile)
+    {
       infile = argv[i];
+      if (strcasestr(infile,".xinp"))
+        if (!args.readXML(infile,false))
+          return 1;
```

Comment:
see Poisson etc and how we re-parse arguments. i know i know, but somebody might want to add more stuff and we want command line to override.